server/routes/index.js

```diff
@@ -24,8 +24,8 @@ router.get("/users/:userid", verify, async (req, res, next) => {
 
   const shop = await shops.findOne(
     { user: user._id },
-    { __v: false, _id: false }
-  );
+    { __v: false, _id: false, user: false }
+  ).populate('items', '-_id -__v');
   console.log("shop is", shop);
```

Comment:
let's create a ticket about using a logging library - we don't need this anymore

Reply:
I will resolve this in the next ticket